geode-connectors/src/main/resources/META-INF/schemas/geode.apache.org/schema/jdbc/jdbc-1.0.xsd

```diff
@@ -46,6 +46,7 @@ XML schema for JDBC Connector Service in Geode.
                 <xsd:attribute type="xsd:string" name="ids" use="optional"/>
                 <xsd:attribute type="xsd:string" name="catalog" use="optional"/>
                 <xsd:attribute type="xsd:string" name="schema" use="optional"/>
+                <xsd:attribute type="xsd:string" name="groups" use="optional"/>
```

Comment:
I'm concerned that you modified the xsd and added "groups" as an attribute to the RegionMapping.
Region does not have such an attribute; and yet regions are created in a group.
What I expected was no change to the xsd or the RegionMapping class.
I thought you would enhance each of the gfsh commands that work with jdbc-mapping to have a --groups option that would scope that command to only look/store in the given group(s).

Reply:
We talked about this, and we think we probably don't need to store this in the cache config. We can remove the code to do so.

Reply:
Agree with @dschneider-pivotal